uber/automated_emails.py

```diff
@@ -14,14 +14,16 @@ class AutomatedEmail:
     }
     extra_models = queries  # we've renamed "extra_models" to "queries" but are temporarily keeping the old name for backwards-compatibility
 
-    def __init__(self, model, subject, template, filter, *, sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True):
+    def __init__(self, model, subject, template, filter, *, sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True, email_id=None):
         self.model, self.template, self.needs_approval = model, template, needs_approval
         self.subject = subject.format(EVENT_NAME=c.EVENT_NAME)
         self.cc = cc or []
         self.bcc = bcc or []
         self.extra_data = extra_data or {}
         self.sender = sender or c.REGDESK_EMAIL
         self.instances[self.subject] = self
+        self.email_id = email_id or self.subject
+        self.email_id = self.email_id.format(EVENT_NAME=c.EVENT_NAME)
```

Comment:
DO we want it to be the same though?  I mean, I get that for backwards-compatibility we need to default it to be based on the subject.  But I don't see anything in the code that's going to blow up if the ident is based on the subject without being == to it.